acquire/acquire.py

```diff
@@ -1504,6 +1504,39 @@ class OSX(Module):
     ]
 
 
+@register_module("--osx-applications-info")
+class OSXApplicationsInfo(Module):
+    DESC = "OS-X info.plist from all installed applications"
+    SPEC = [
+        ("glob", "/Applications/*/Contents/Info.plist"),
+        ("glob", "Applications/*/Contents/Info.plist", from_user_home),
+    ]
+
+
+@register_module("--osx-applications-data")
+class OSXApplicationsData(Module):
+    DESC = "OS-X data from all installed applications"
+
+    @classmethod
+    def get_spec_additions(cls, target: Target, cli_args: argparse.Namespace) -> Iterator[tuple]:
+        spec = set()
+
+        for app_support_path in itertools.chain(
+            ["/Library/Application Support"],
+            from_user_home(target, "Library/Application Support"),
+        ):
+            for path in target.fs.path(app_support_path).glob("*/*/*"):
+                if not path.is_file():
+                    continue
+
+                if path.stat().st_size >= (1 * 1024 * 1024):  # 1MB
+                    continue
```

Comment:
Hereby the suggestion code ;)

```suggestion
            for path in target.fs.path(app_support_path).rglob("*"):
                if not path.is_file():
                    continue
               
               if path.parts[1] == "Library" and len(path.parts) > 5:
                    continue

               if len(path.parts) > 7:
                    continue

               if path.stat().st_size >= (1 * 1024 * 1024):  # 1MB
                    continue
```

Reply:
The suggested code takes `3m49.462s` compared to `5.232s` [`ae3ec77` (#96)](https://github.com/fox-it/acquire/pull/96/commits/ae3ec772eb3895f8194527727f89035cfbd797f3), but the code looks a bit ugly.

```
[...]
   # Special case for the non-user directory "/Library/Application Support/"
                if path.parts[1] == "Library" and len(path.parts) > 6:
                    continue

                if len(path.parts) > 8:
                    continue
[...]
```